app/helpers/api/vulnerabilities_helper.rb

```diff
@@ -37,11 +37,29 @@ def bdsa_vulnerability_age(start_date)
   end
 
   def bdsa_references(data)
-    return unless data
+    references = bdsa_reference_icons.keys.each_with_object({}) { |link, memo| memo[link] = [] }
+    data.each do |link|
+      type = bdsa_reference_type(link['type'])
+      next unless references[type]
+
+      references[type] << link['href']
+    end
+    references.reject { |_type, links| links.empty? }
+  end
 
-    references = data.select { |link| %w[ADVISORY VENDOR_UPGRADE].include?(link['type']) }
-    references.each_with_object({}) do |link, memo|
-      memo[link['type']] = (memo[link['type']] || []).push(link['href'])
+  def bdsa_reference_type(type)
+    case type
+    when 'LINK' then 'PATCH'
```

Comment:
I don't believe this is correct.  Link should be in a category called Other.  POC should be left as POC.  Patch should remain as Patch.